lynx/common/enitity.py

```diff
@@ -1,6 +1,33 @@
+from __future__ import annotations
+from dataclasses import dataclass
+
 from lynx.common.serializable import Serializable
 
+
 # Right now it just represents common root
 # for both `Object` and `Action`
 class Entity(Serializable):
-    pass
\ No newline at end of file
+    @dataclass
+    class Exported(Serializable):
+        type: str = ""
+        attributes: str = ""
+
+    def get_type(self) -> str:
+        return type(self).__name__
+
+    def get_attributes(self) -> str:
+        return super().serialize()
+
+    def serialize(self) -> str:
+        return self.Exported(self.get_type(), self.get_attributes()).serialize()
+
+    @classmethod
+    def deserialize(cls, json_string: str) -> Entity:
+        from lynx.common.actions.move import Move
```

Comment:
please add comment to this imports, why are they here, as they are confusing at first

Reply:
I agree with @alpinus4

Reply:
@alpinus4 @Krystian030 I will add one.